### DISCUSSION TITLE
Turn screen on by Navigation instructions - less than 750 meter default setting / screen off just after turn

### DISCUSSION THREAD

**Discussion by IronParticle** [osmandapp/OsmAnd#14080] [Ideas]
I use the Navigation instruction to turn the screen on for my cycling trips. It seems to turn the screen on at a default distance of about 750 meters to the turn. For my taste this is way to early. For a cycling profile this could be either less by default (250 meters perhaps) or set by the user. An alternative could be to let it be relative to current speed. When I'm barrelling downhill at 60 km/h maybe 750 meters is fine, but at 25 km/h 250 meters should do I think. 

Also the screen could be turned off automatically AFTER passing the navigating turn. Either in seconds or meters. 
Basically what I would prefer is to have the screen turned on in time to navigate the turn (but less than 750 meters) and have the screen turn off just after successfully passing that navigating turn.

**Comment by Lee-Carre** [comment]
I think this is (somewhat) configurable, including based on measured speed as you describe.

Have a look in the profile config, under routing options (in the navigation section, if I recall); you'll want to tune your `default speed` (to closely match your typical average moving speed), and there's a slider for adjusting when announcements (e.g. of next turn) are made (sounds like you want much earlier than default).

**Comment by IronParticle** [comment]
I can only find an Announcement time under Voice prompts. I don't actually use sound as it isn't really practical on a racing bike, but if this also sets the screen on later when nearing a turn then it is what I was looking for. But I reckon that doesn't work like that, especially when the Voice prompts are turned off. Haven't been able to test this yet.
